util/vizerror/vizerror.go

```diff
@@ -12,7 +12,8 @@ import (
 
 // Error is an error that is safe to display to end users.
 type Error struct {
-	err error
+	err     error
+	wrapped error
```

Comment:
err error // internal
wrapped error // visible to users

Reply:
I've added comments, though note that it's the opposite.